src/main/java/be/seeseemelk/mockbukkit/MockUnsafeValues.java

```diff
@@ -276,15 +282,32 @@ public boolean isSupportedApiVersion(String apiVersion)
 	@Override
 	public byte[] serializeItem(ItemStack item)
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		final ByteArrayOutputStream bao = new ByteArrayOutputStream();
```

Comment:
```suggestion
		Preconditions.checkNotNull(item, "null cannot be serialized");
		Preconditions.checkArgument(item.getType() != Material.AIR, "air cannot be serialized");
		final ByteArrayOutputStream bao = new ByteArrayOutputStream();
```

Reply:
... Well AIR can  be serialized without issue, the question is if should be...

Reply:
I copied this directly from the paper implementation, I don't think it's possible.